tests/status_response/test_java.py

```diff
@@ -99,3 +101,555 @@ class TestJavaStatusVersion(BaseStatusResponseTest):
+                {"modid": "terralith", "modmarker": "0.0NONE"},
+                {"modid": "mininggadgets", "modmarker": "1.11.0"},
+                {"modid": "crafttweaker", "modmarker": "9.1.197"},
+                {"modid": "akashictome", "modmarker": "1.5-20"},
+                {"modid": "forge", "modmarker": "ANY"},
+                {"modid": "colossalchests", "modmarker": "1.8.3"},
+                {"modid": "selene", "modmarker": "1.18.2-1.17.9"},
+                {"modid": "drippyloadingscreen", "modmarker": "1.6.4"},
+                {
+                    "modid": "craftingtweaks",
+                    "modmarker": "<not required for client>",
+                },
+                {"modid": "minecraft", "modmarker": "1.18.2"},
+                {"modid": "terrablender", "modmarker": "1.18.2-1.1.0.102"},
+                {
+                    "modid": "sophisticatedbackpacksvh",
+                    "modmarker": "1.18.2-1.0.4.12",
+                },
+                {"modid": "mousetweaks", "modmarker": "ANY"},
+                {"modid": "titanium", "modmarker": "3.5.6"},
+                {"modid": "jade", "modmarker": "<not required for client>"},
+                {"modid": "createtweaker", "modmarker": "2.0.0.17"},
+                {"modid": "easy_villagers", "modmarker": "1.18.2-1.0.10"},
+                {"modid": "pipez", "modmarker": "1.18.2-1.1.5"},
+                {"modid": "iceberg", "modmarker": "ANY"},
+                {"modid": "flywheel", "modmarker": "<not required for client>"},
+                {"modid": "mantle", "modmarker": "1.9.27"},
+                {"modid": "ecologics", "modmarker": "1.7.3"},
+                {"modid": "quark", "modmarker": "3.2-358"},
+                {"modid": "xaerominimap", "modmarker": "22.11.1"},
+                {"modid": "pigpen", "modmarker": "8.0.1"},
+                {"modid": "fastbench", "modmarker": "6.0.2"},
+                {"modid": "polymorph", "modmarker": "1.18.2-0.44"},
+                {"modid": "autoreglib", "modmarker": "1.7-53"},
+                {"modid": "storagedrawers", "modmarker": "10.2.1"},
+                {"modid": "fluxnetworks", "modmarker": "7.0.7.8"},
+                {"modid": "neoncraft2", "modmarker": "2.2"},
+                {"modid": "enercell", "modmarker": "0.0NONE"},
+                {"modid": "appleskin", "modmarker": "2.4.0+mc1.18"},
+                {
+                    "modid": "ferritecore",
+                    "modmarker": "<not required for client>",
+                },
+                {"modid": "modularrouters", "modmarker": "9.1.1-93"},
+                {"modid": "refinedstorageaddons", "modmarker": "0.8.2"},
+                {"modid": "openloader", "modmarker": "12.0.1"},
+                {"modid": "the_vault", "modmarker": "1.18.2-2.0.10.869"},
+            ],
+        ),
+        ("truncated", False),
+    ]
+
+    @fixture(scope="class")
+    def build(self):
+        return ForgeData.build(
+            RawForgeData(
```

Comment:
There should also be a test for the older response, where `d` key wasn't yet a thing.

Reply:
Added in 626854f